database/src/schema/permission_group.rs

```diff
@@ -1,27 +1,37 @@
 use chrono::{DateTime, Utc};
 use cosmicverge_shared::permissions::{Permission, Service};
 
+#[derive(Debug)]
 pub struct PermissionGroup {
     pub id: i32,
     pub name: String,
     pub created_at: DateTime<Utc>,
 }
 
-pub struct Statement {
-    pub id: i32,
-    pub service: String,
-    pub permission: Option<String>,
-    pub created_at: DateTime<Utc>,
-}
-
 impl PermissionGroup {
-    pub async fn create<'e, E: sqlx::Executor<'e, Database = sqlx::Postgres>>(
-        name: String,
+    pub async fn create<
+        'e,
+        E: sqlx::Executor<'e, Database = sqlx::Postgres>,
```

Comment:
On a side-note, we should definitely improve this pattern, instead of passing the executor all the time, we should rly make a `trait` on executor with all these functions, or better yet, wrap the executor in a `struct` and put all functionality into methods.